src/components/Lobby/index.js

```diff
@@ -178,8 +205,8 @@ export default function Lobby() {
         <LeftContainer>
           <RoomsContainer>
             <RoomsLists>
-              {rooms &&
-                rooms.map((roomData) => (
+              {roomsList &&
```

Comment:
```suggestion
              {roomsList.length &&
```
`[]` 은 `Boolean`이 아니니 `length`라는 속성을 사용해 배열이 비었는지 아닌 지에 대해 확인을 해주는 것이 맞아보입니다.